android/src/main/java/it/innove/BleManager.java

```diff
@@ -3,6 +3,7 @@
 import static android.app.Activity.RESULT_OK;
 import static android.bluetooth.BluetoothProfile.GATT;
 import static android.os.Build.VERSION_CODES.LOLLIPOP;
+import static android.os.Build.VERSION_CODES.UPSIDE_DOWN_CAKE;
```

Comment:
@marcosinigaglia @lucaswitch doesn’t this break when API level < 34?

Reply:
I think it is not a problem as discussed in here: https://stackoverflow.com/questions/21874227/why-do-android-os-build-version-codes-work-on-older-platforms